### ISSUE TITLE
Mockup for Gradience main website

### ISSUE THREAD

**Issue by tfuxu** [GradienceTeam/Design#15]
We should redesign our main website to make it more GNOME-y.
As of now it looks a little bit bland with that default template that we use in Lume, and there's not really any content there except for the `How to contribute?` section, with a footer under it, which consists of a link to Discord server and the Gradience main repo.
It would be nice to have a main page that looks more like the page of some GNOME project, like what we've done in [gradienceisahack.github.io](https://gradienceisahack.github.io/) website.

A great inspirations in my opinion for this redesign would be [forty.gnome.org](https://forty.gnome.org/) and [repainter.app](https://repainter.app/) (mentioned by daudix some time ago).

**Comment by 0xMRTT** [comment]
That's a nice idea. I would like to mention that repainter.app isn't open source and the forty.gnome.org to (maybe i'm wronge for the gnome one ?). I would also mention https://collision.geopjr.dev/

**Comment by tfuxu** [comment]
> That's a nice idea. I would like to mention that repainter.app isn't open source and the forty.gnome.org to (maybe i'm wronge for the gnome one ?).

I mean, we're not going to copy them one to one, but we can get some inspiration from how they look, to slightly influence the final look of our page. 
Also about forty.gnome.org website, almost every GNOME project site has a note at the bottom, that says it is distributed under the `Creative Commons CC-BY-SA` license, so if we based our website on that page, we would just have to give a credit in the footer, such as _This page was based on ..._ or something like that, just like we did on gradienceisahack page.

**Comment by tfuxu** [comment]
> I would also mention https://collision.geopjr.dev/ and https://libadwaita.geopjr.dev


Interesting suggestion! Geopjr sites have always looked very nice to me, and they are pretty creative in their look, so we could also inspire our site on them.

**Comment by 0xMRTT** [comment]
Yes :+1:

**Comment by tfuxu** [comment]
What is the current progress on this mockup? Are there perhaps any drafts already created?

**Comment by daudix-UFO** [comment]
I have nothing :/

**Comment by tfuxu** [comment]
Okay, no problem

**Comment by daudix-UFO** [comment]
I will start making one right now 😉

**Comment by daudix-UFO** [comment]
Current state of mockup (WIP)

![preview](https://user-images.githubusercontent.com/77155297/203670207-b437cc6a-d336-40c6-87a3-a291d09fc372.png)

**Comment by tfuxu** [comment]
Wow, already looks really good!

**Comment by daudix-UFO** [comment]
Added WIP mockup

https://github.com/GradienceTeam/Design/raw/main/Mockups/web/gradience/gradience-wip.png

**Comment by 0xMRTT** [comment]
I like it, what about adding an "explore more theme" in the "wide range of palettes" section ? Which will redirect to the explore page ?

**Comment by daudix-UFO** [comment]
![Screenshot_20221124-210913_Firefox~2](https://user-images.githubusercontent.com/77155297/203845965-b360c659-4eb3-4b8a-8c02-e4acc92df5c6.png)

**Comment by 0xMRTT** [comment]
I mean ...

**Comment by 0xMRTT** [comment]
![IMG_20221125_182536.jpg](https://user-images.githubusercontent.com/105598867/204032927-8e763073-2471-4442-b71c-c9ca885b7c95.jpg)

Make the highlighted section scrollable horizontally and show at the end a see more theme button ?

**Comment by daudix-UFO** [comment]
I like the idea!

**Comment by daudix-UFO** [comment]
Basically this (https://repainter.app)

![image](https://user-images.githubusercontent.com/77155297/204053907-e151e32c-11d6-426a-a2b8-147643b1deae.png)

**Comment by daudix-UFO** [comment]
BTW, I like how theme colors are presented

**Comment by daudix-UFO** [comment]
Moved mockup from WIP status

![gradience](https://github.com/GradienceTeam/Design/raw/main/Mockups/web/gradience/gradience.png)

**Comment by daudix-UFO** [comment]
I think the mockup is in good shape now, any preferences on adding/changing something?

@tfuxu @0xMRTT

**Comment by tfuxu** [comment]
Maybe we could instead of a black background, use a subtle gradient (something like in https://usebottles.com), or an image under slightly transparent background, for which I could create a [parallax](https://www.sketch.com/blog/what-is-a-parallax-effect/) effect in CSS.

**Comment by daudix-UFO** [comment]
Hmm, idea with paralax is nice, not sure about what image we can use

**Comment by tfuxu** [comment]
I thought initially about [Pretty Purple](https://github.com/GradienceTeam/Design/blob/main/Backgrounds/pretty-purple.png) background, but I think that it is too dark in contrast to the rest of the webpage.

**Comment by daudix-UFO** [comment]
Hmm, actually why not

**Comment by tfuxu** [comment]
Actually, it looks quite nice
![gradience-proto5](https://user-images.githubusercontent.com/73042332/204106915-0614e71b-f412-4efc-ad6d-85a0b503b5a5.png)

**Comment by daudix-UFO** [comment]
![image](https://user-images.githubusercontent.com/77155297/204106932-c51df080-0acf-4471-92b8-90547df9366d.png)

**Comment by daudix-UFO** [comment]
More than nice

**Comment by daudix-UFO** [comment]
Slight black tint (20%) makes it even better

**Comment by daudix-UFO** [comment]
![tint](https://user-images.githubusercontent.com/77155297/204107054-97019840-14ff-4dd7-a4df-e3ab6ea91aa5.png)

**Comment by daudix-UFO** [comment]
With/Without tint

**Comment by tfuxu** [comment]
Hmm, without the blur it looks better IMO.
I'm leaving it as-is in PR for now, because I don't know yet how to do those filters for the background images.

**Comment by daudix-UFO** [comment]
OK

**Comment by 0xMRTT** [comment]
I like the blur but maybe less Blur will be better ?

**Comment by daudix-UFO** [comment]
Hmm, not sure about readability

**Comment by daudix-UFO** [comment]
10% / 15% blur

![10](https://user-images.githubusercontent.com/77155297/204113278-952d8d3b-a0b6-4722-8fbe-06110559928e.png)

![15](https://user-images.githubusercontent.com/77155297/204113279-03010d20-d711-421c-9077-6d7783f83f58.png)

**Comment by 0xMRTT** [comment]
And what about light/dark mode ?

**Comment by daudix-UFO** [comment]
Yeah, it pretty useless in our case

**Comment by daudix-UFO** [comment]
So, what amount of blur you like?

**Comment by 0xMRTT** [comment]
10% ?

**Comment by tfuxu** [comment]
Yea, 10% should be alright

**Comment by daudix-UFO** [comment]
OK

**Comment by daudix-UFO** [comment]
BTW, laptop's shell blur is 15%

**Comment by tfuxu** [comment]
We should also change colors for 'Get Gradience' button, because it's currently slightly hard to look at, and it is completely disqualified from the [WCAG](https://www.w3.org/WAI/standards-guidelines/wcag/) standard test.
![Screenshot from 2022-11-27 01-42-35](https://user-images.githubusercontent.com/73042332/204114261-0ba07a36-8b44-4802-a653-fb83cdce87be.png)

**Comment by daudix-UFO** [comment]
Good catch, I will try to find color that look good and have good contrast

**Comment by daudix-UFO** [comment]
Alright, should be good now

![image](https://user-images.githubusercontent.com/77155297/204114506-f5ef343f-a308-4ff3-bc8c-ccab64a8d6a3.png)
![image](https://user-images.githubusercontent.com/77155297/204114504-1d79fc79-3d04-4675-b1e9-6aa0e88f2ff5.png)

**Comment by daudix-UFO** [comment]
Can we make the paralax effect in header more like this?

https://codepen.io/robatronbobby/pen/vYjQZNa

_[See "Method 2" here](https://blog.logrocket.com/create-parallax-scrolling-css)_

**Comment by tfuxu** [comment]
I can do that, but what image should I use for this instead of the current one?

**Comment by tfuxu** [comment]
Okay, I replaced it with SVG version, but I'm not sure about rotating it. I don't think it would look good.

**Comment by daudix-UFO** [comment]
I can try to make square version

**Comment by tfuxu** [comment]
Why a square?

**Comment by daudix-UFO** [comment]
To be compatible with mobile and desktop

**Comment by tfuxu** [comment]
But it already is, and on desktop it might be too stretched on higher resolutions.
